### PR TITLE
Add React and Webpack to App

### PR DESCRIPTION
**Pull request by katebee**
😄 

Basic version of app, that will build and run, including React components

picture 211

### CODE REVIEW

**Comment by tsop14** [comment]
Can we have a screen shot? Just for the hype!

File: app/AppComponents.scala [hunk 1]
```diff
@@ -29,8 +29,8 @@ trait AWSComponent { self: BuiltInComponents =>
 
 trait ControllersComponent {
   self: BuiltInComponents with AWSComponent =>
-  def messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
-  def appController = new Application(dynamoClient, talksTableName)
+  val messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
```
Comment (tsop14) [inline app/AppComponents.scala]: Why did you change these to `val`s?

Comment (susiecoleman) [reply app/AppComponents.scala]: Originally we changed them out of desperation as we couldn't work out what had broken but they should be def (or lazy val) 😃

Comment (tsop14) [reply app/AppComponents.scala]: Yeah @marialivia16 and I had a problem and concluded these should be `def`s or `lazy val`s.

**Comment by Calum-Campbell** [comment]
👍  Awesome !

File: app/views/Application/app.scala.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@(title: String, jsLocation: String)
```
Comment (marialivia16) [inline app/views/Application/app.scala.html:1]: Any reason for which you created a new folder under `views`. Do we really need this extra level of nesting?

Comment (katebee) [reply app/views/Application/app.scala.html:1]: No real reason, sorry! Please feel free to flatten it out. 👍

Comment (marialivia16) [reply app/views/Application/app.scala.html:1]: Thanks! 👍
